binary_policies.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Policies for Binary Packages
+layout: single
+author_profile: false
+---
+
+Spack has the ability to create, download, install, and bootstrap
+dependencies from binary packages. Spack signs binary packages with
+gpg, and the ``spack gpg trust`` or ``spack buildcache keys -it``
+commands tell Spack explicitly which gpg keys to trust.
+
+When bootstrapping the lowest-level Spack dependencies from binaries,
+binary verification is more complicated. Spack will bootstrap
+``clingo`` and ``gpg`` without access to its normal gpg-based
+verification capabilities. In those cases, Spack will bootstrap from a
+very limited selection of binaries with sha256 checksums associated in
+Spack. These bootstrapping binaries will also be associated with a
+binary installation policy.
```

Comment:
"binary installation policy" isn't defined yet -- we should probably lead with some motivation and that definition. Why is trusting binaries special (vs sources), what are Spack's policies -- basically the idea that a signature or a set of binaries are created using some process, and we want users to understand the process and trust it (or not)